chef/cookbooks/hyperv/recipes/openstack_install_ceilometer.rb

```diff
@@ -12,7 +12,7 @@
   rmdir /S /Q #{node[:openstack][:location]}\\dist
   ren #{node[:openstack][:location]}\\#{node[:openstack][:ceilometer][:name]}-#{node[:openstack][:ceilometer][:version]} #{node[:openstack][:ceilometer][:name]}
   EOH
-  not_if { ::File.exists?("#{node[:openstack][:location]}\\#{node[:openstack][:ceilometer][:name]}") }
+  not_if { ::File.exist?("#{node[:openstack][:location]}\\#{node[:openstack][:ceilometer][:name]}") }
```

Comment:
Metrics/LineLength: Line is too long. [101/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)